oci/loki/documentation.yaml

```diff
@@ -38,5 +42,5 @@ debug:
     To get an interactive shell:
 
     ```bash
-    docker exec -it loki-container /bin/bash
+    docker exec -it loki-container exec /bin/bash
```

Comment:
Also worth having a `# <comment>` to highlight the `exec` Pebble command, which is not needed for pre-ROCKs images